docs/configuration.md

```diff
@@ -87,25 +87,45 @@ Supported placeholders/variables are:
  1. **{locale\_with\_underscore}** same as above except all hyphens '-' will be replaced with underscores '_'. This is typically used in properties and gettext projects.
  1. **{extension}** the source document file extension
 
-For example, given a source document is found at `/home/user/myproject/src/main/resource/message.properties` 
+For example, if your file structure is like following (where `{projectRoot}` is the root of your project and where zanata.xml lives):
 
-... and your source document root (src-dir option) is set to "." 
-
-... Your zanata.xml is located at `/home/user/myproject` 
+```
+{projectRoot}/
+              templates/messages/kdeedu/kalzium.pot
+              templates/messages/kdeedu/artikulate.pot
+              de-DE/messages/kdeedu/kalzium.po
+              de-DE/messages/keeedu/artikulate.po
+              ...
+              zanata.xml
+```
 
-... For a locale mapping defined as `<locale map-from="zh-CN">zh</locale>`: 
+Here we have two "pot" files as source documents and two "po" files as "de-DE" language translation documents.
```

Comment:
Suggest:
Here we have two source documents (with "pot" extension) and two translation documents (with "po" extension) for the locale "de-DE".